src/capture.js

```diff
@@ -389,6 +389,13 @@ Capture.isIOS8_0 = function() {
     return IOS8_REGEX.test(window.navigator.userAgent);
 };
 
+var IOS_REGEX = /ip(?:hone|od|ad).*Version\/(\d){1,2}\.\d/i;
+Capture.isIOS8OrGreater = function() {
+    var match = window.navigator.userAgent.match(IOS_REGEX);
+
+    return match ? match[1] >= 8 : false;
```

Comment:
Could we add a test for this? Would be good to assert that this returns true for an iOS8 and 9 UA, and false for an iOS7.